homeassistant/components/insteon_hub.py

```diff
@@ -69,30 +69,30 @@ def setup(hass, config: dict) -> bool:
 class InsteonDevice(Entity):
     """Represents an insteon device."""
 
-    def __init__(self: Entity, node: object) -> None:
```

Comment:
We don't have to remove it - I was told I was being over zealous by typing `self`